src/scripts/main.js

```diff
@@ -354,7 +354,24 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+people.forEach((person) => {
+  const row = document.createElement('tr');
 
-// write your code here
+  const data = [
+    person.name,
+    person.sex === 'm' ? 'Male' : 'Female',
+    person.born,
+    person.died,
+    person.died - person.born,
+    Math.ceil(person.died / 100),
+  ];
+
+  data.forEach((value) => {
+    const cell = document.createElement('th');
+
+    cell.textContent = value;
+    row.appendChild(cell);
+  });
+
+  document.querySelector('.dashboard').appendChild(row);
```

Comment:
Ensure that there is an element with the class 'dashboard' in your HTML. If it doesn't exist, this line will cause a runtime error.